contracts/NFTMarketplace.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import '@openzeppelin/contracts/utils/Counters.sol';
+import './MarketItem.sol';
+
+contract NFTMarketplace {
+    MarketItem private marketItem;
+    using Counters for Counters.Counter;
+    Counters.Counter private _collectionIds;
+    uint256 public LISTING_FEE = 0.0001 ether;
+    address payable private _marketOwner;
+
+    mapping(uint256 => Collection) public collectionLedger;
+    mapping(uint256 => MarketNft) public nftLedger;
+    mapping(address => uint256) private credits;
+
+    struct Collection {
+        uint256 collectionId;
+        string seller;
```

Comment:
@ymincheva `seller` is not needed
you need `collectionName` and eventually `collectionDescription`